lib/pure/times.nim

```diff
@@ -700,46 +700,53 @@ else:
     result.inc tm.hour * secondsInHour
     result.inc tm.minute * 60
     result.inc tm.second
+  
+  when not defined(windows):
+    # Trick to find the real size of time_t
+    proc sizeof(typ: typedesc[Time]): int =
+      {.emit: [result, " = sizeof(time_t);"].}
```

Comment:
Is this the best way to do this? The standard `sizeof` can't be used because `time_t` is always declared as `clong` by the posix module.